frontend/src/components/clinics/components/add-clinic-popup/add-clinic-popup.tsx

```diff
@@ -30,15 +34,26 @@ const AddClinicPopup: React.FC<IProps> = ({
   onCreateClinic,
   isOpen,
 }) => {
+  const [image, setImage] = React.useState<string>(DEFAULT_CLINIC_VALUE.imagePath);
```

Comment:
We don't want an additional state for it
Do it like in the `frontend/src/components/common/edit-user-popup/edit-user-popup.tsx`